samples/msgext-ai-doc-compliance-checker/README.md

```diff
@@ -0,0 +1,130 @@
+---
+page_type: sample
+description: This sample demonstrates integrating Azure Open AI into a Teams message extension to check a contract proposal's compliance with a policy checklist. It works with Teams Toolkit for Visual Studio Code and JavaScript and can be used in Copilot for Microsoft 365.
+products:
+- office-teams
+- copilot-m365
+- azure open-ai
+languages:
+- javascript
+---
+
+# Compliance Checker using Azure Open AI
+
+This sample uses Azure Open AI to compare proposal documents uploaded to an Azure Blob storage with a policy guideline that is also uploaded in the blob.
+ 
```

Comment:
Mention how to modify this sample to verify other documents.

Also, mention about how many checklist files we can verify against one documentation?

Reply:
Engineering team has provided only the content.

Reply:
I have added below:
**Note: This sample is configured to check documents compliance based on only one checklist, so you can modify the code as per your requirements.**